tests/unit/conftest.py

```diff
@@ -138,3 +147,34 @@ def mock_airflow_connection_datarobot_basic_credentials(mocker, dr_basic_credent
         ),
     )
     mocker.patch.dict("os.environ", AIRFLOW_CONN_DATAROBOT_BASIC_CREDENTIALS_DEFAULT=conn.get_uri())
+
+
+@pytest.fixture(scope="function", autouse=True)
```

Comment:
```suggestion
@pytest.fixture(autouse=True)
```

`scope="function"` is the default so you don't need to specify that.

Reply:
You also have a lot of fixtures that have `autouse=True`. That's typically a code smell and having them here in such a high level `conftest.py` can end up being very surprising to test writers in the future (e.g. methods are patched that they didn't expect to be w/o having read through the entire contents of this file to understand all the places that are mocked out).

Are these fixtures really so essential or can they only be active in specific tests?